inferno-vc/src/Inferno/VersionControl/Operations/Filesystem.hs

```diff
@@ -204,7 +218,7 @@ instance (MonadIO m, MonadMask m, AsType VCStoreError err) => InfernoVCOperation
       createDirectoryIfMissing True $ storePath </> "removed" </> "deps"
 
     withWrite lock $ do
-      (metas :: [VCMeta Value Value VCObjectHash]) <- fetchVCObjectHistory obj_hash
+      metas <- fetchVCObjectHistory obj_hash
```

Comment:
ditto This change is made possible because deleteAutosavedVCObject now knows what g and a are